Framework/Algorithms/src/ReflectometryReductionOneAuto2.cpp

```diff
@@ -292,6 +297,20 @@ void ReflectometryReductionOneAuto2::init() {
   setPropertyGroup("Rho", "Polarization Corrections");
   setPropertyGroup("Alpha", "Polarization Corrections");
 
+  // Flood correction
+  propOptions = {"Workspace", "ParameterFile"};
+  declareProperty("FloodCorrection", "Workspace",
+                  boost::make_shared<StringListValidator>(propOptions),
+                  "The way to apply flood correction: "
+                  "Workspace - use FloodWorkspace property to get the flood "
+                  "workspace, ParameterFile - use parameters in the parameter "
+                  "file to construct and apply flood correction workspace.");
+  declareProperty(
```

Comment:
Be aware that you may break existing user scripts since the new properties are not added at the end, i.e. after the debug properties?

Reply:
Is it a real issue? I would prefer that more important properties go before less important ones. And input properties to go before the output ones.